.github/workflows/build_downstream_deps.yml

```diff
@@ -102,10 +102,13 @@ jobs:
              printf "\npath = \"../cedar/cedar-policy\"" >> Cargo.toml
              cargo build
              cargo test
-      - name: build tinytodo
+      - name: echo
         working-directory: cedar-examples/tinytodo
-        run: |
-             printf "\npath = \"../cedar/cedar-policy\"" >> Cargo.toml
-             cargo build
-             cargo test
+        run: echo "'${{ needs.get-branch-name.outputs.branch_name }}'"
+
+  build-tiny-todo:
+    needs: get-branch-name
+    uses: cedar-policy/cedar-examples/.github/workflows/build_tiny_todo_reusable.yml@main
+    with:
+      cedar_policy_ref: ${{ github.head_ref }}
```

Comment:
Since this is for `3.0`, I think we'll eventually want `cedar_examples_ref: "releases/3.0"` or similar. For now, I think we should do `cedar_examples_ref: "main"` to make it obvious what to change.